File: hw/vargas/u1/hw11ChickenCoops/ChickenFarmSimulatorPY/Egg_model.py
class Egg:
    
    def __init__(self, size):
        self._size = size

    def __str__(self):
        return f"Egg{{size={self._size}}}"

    def get_size(self):
        return self._size

    def set_size(self, size):
        self._size = size

File: hw/vargas/u1/hw08FileManagement/FileManagmentJS/Egg_model.js
export class Egg {
    constructor(size) {
        this._size = size;
    }

    getSize() {
        return this._size;
    }
    setSize(size) {
        this._size = size;
    }

    toString() {
        return `Egg{size=${this._size}}`;
    }
}